### ISSUE TITLE
Webinterface Sound Controls not working with USB Sound 

### ISSUE THREAD

**Issue by svo08** [MiczFlor/RPi-Jukebox-RFID#20]
Hi,

first, thanks for that great project :-)

Im using a external USB Soundcard (Soundblaster Play).
I did manage to get the Card working.

Everything is working fine except the volume control via webinterface.

I discoverd that i have to use different command to adjust Volume
amixer -D hw:U0x41e0x30d3 sset Speaker 100%
Think it is due to the command amixer sset ´PCM´ 100% is not working.
I was able to adjust the Control Script for RFID but i do not know what and where i have to adjust the files for the webinterface to get it to work there as well.
Think it has to be adjustet like i did it in the control script.
Can you please help me.

Thanks

**Comment by svo08** [comment]
Sorry, done i did found it index.php.
I adjusted the line and volume control is working.

**Comment by MiczFlor** [comment]
@svo08 can you please post your solution here, so I can include it in the code. I was thinking of adding this for HiFiBerry anyway. So this could be a good addition.

**Comment by svo08** [comment]
Hi, not sure if it is possible to add the volume command to config file.
Then there will be a single place, single command to replace.

As i wrote in the post im using a different sound card (USB One).
I was able to get it work with description in the docs.
$ cat /proc/asound/modules 
There is a way to adress with amixer the HW Card.
So "amixer sset ´PCM´ 100%" was not working but adressing sound card with 
"amixer -D hw:U0x41e0x30d3 sset Speaker 100%"

So i had to replace the string amixer sset ´PCM´ with "amixer -D hw:U0x41e0x30d3 sset Speaker"
I did this in index.php in htdocs and in the control script rfid_trigger_play.sh
